src/app/cart/cart.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="cart--item-list-container">
+  <ng-container *ngIf="this.getBindedChosenItems().length !== 0">
+    <ul class="item-list cart--item-list">
+      <li *ngFor="let item of this.getBindedChosenItems()">
+        <img
+        class="cart--item-icon"
+        src="{{'../../assets/icons/' + item.id + '.svg'}}"
```

Comment:
shouldn't need to add a relative path to the image here. We could just do `'assets/icons/' + item.id + '.svg'`